commands/status.go

```diff
@@ -39,6 +39,7 @@ PMM Client:
 	Time drift       : {{ .PMMAgentStatus.ServerClockDrift }}
 	Latency          : {{ .PMMAgentStatus.ServerLatency }}{{ end }}
 	pmm-admin version: {{ .PMMVersion }}
+	pmm-agent version: {{ .PMMAgentStatus.AgentVersion }}
```

Comment:
It will be empty for older pmm-agent. I think we should add "unknown" or something like that

Reply:
Changed to "unknown".